base/mpfr.jl

```diff
@@ -26,6 +26,9 @@ import Base.Math.lgamma_r
 
 import Base.FastMath.sincos_fast
 
+const MPFR_VERSION = VersionNumber(unsafe_string(ccall((:mpfr_get_version,:libmpfr), Ptr{Cchar}, ())))
+const MPFR_PATCHES = split(unsafe_string(ccall((:mpfr_get_patches,:libmpfr), Ptr{Cchar}, ())),' ')
```

Comment:
I was just copying what GMP does. But yes, maybe we should.

The other issue is naming: LibGit2 simply names it `VERSION` (i.e. `LibGit2.VERSION`). Should we use this for consistency?